Core/Core/API/APICourseRequestable.swift

```diff
@@ -155,3 +155,17 @@ struct PostCourseRequest: APIRequestable {
         return "\(ContextModel(.account, id: accountID).pathComponent)/courses"
     }
 }
+
+// https://canvas.instructure.com/doc/api/courses.html#method.courses.users
+public struct GetCourseUsersRequest: APIRequestable {
```

Comment:
Do we need this if we have the `GetContextUsersRequest`?

Reply:
Ah nope. This was old news. And that explains the bad path. I'll remove it. Thanks.